apps/ledger-live-mobile/src/hooks/useSetNavigationHeader.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useMemo, useEffect } from "react";
+import { useNavigation } from "@react-navigation/native";
+
+import { navigationRef } from "../rootnavigation";
+
+export type Args = {
+  headerShown?: boolean;
+  title?: string;
+  HeaderLeft?: React.ComponentType | null;
+  HeaderRight?: React.ComponentType | null;
```

Comment:
I'm not sure that React.ComponentType works for these, for instance it might break with class components as react-navigation will try to call it as a function (and not with `React.createElement(headerWhatever)`).
In my opinion let's follow the typing of react-navigation here

Reply:
done ;)